src/main/java/com/twitter/http2/DefaultHttp2Headers.java

```diff
@@ -0,0 +1,65 @@
+package com.twitter.http2;
+
+import io.netty.handler.codec.DefaultHeaders.NameValidator;
+import io.netty.handler.codec.http.DefaultHttpHeaders;
+import io.netty.util.AsciiString;
+import io.netty.util.ByteProcessor;
+import io.netty.util.internal.PlatformDependent;
+
+import static io.netty.util.internal.ObjectUtil.checkNotNull;
+
+public class DefaultHttp2Headers extends DefaultHttpHeaders {
+  private static final ByteProcessor HEADER_NAME_VALIDATOR = new ByteProcessor() {
+    @Override
```

Comment:
No it rejects ':' in the key names

On Saturday, October 24, 2015, Madhu Palani <notifications@github.com>
wrote:

> In src/main/java/com/twitter/http2/DefaultHttp2Headers.java
> <https://github.com/twitter/netty-http2/pull/9#discussion_r42939995>:
>
> > @@ -0,0 +1,65 @@
> > +package com.twitter.http2;
> > +
> > +import io.netty.handler.codec.DefaultHeaders.NameValidator;
> > +import io.netty.handler.codec.http.DefaultHttpHeaders;
> > +import io.netty.util.AsciiString;
> > +import io.netty.util.ByteProcessor;
> > +import io.netty.util.internal.PlatformDependent;
> > +
> > +import static io.netty.util.internal.ObjectUtil.checkNotNull;
> > +
> > +public class DefaultHttp2Headers extends DefaultHttpHeaders {
> > +  private static final ByteProcessor HEADER_NAME_VALIDATOR = new ByteProcessor() {
> > +    @Override
>
> Doesn't all of this come for free when we use super(true)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twitter/netty-http2/pull/9/files#r42939995>.
>